src/drawerbase.js

```diff
@@ -52,6 +52,9 @@
  *   for all drawers created and used for rendering, particularly the main viewer drawer. However,
  *   if you need to use particular viewer API for rendering an offscreen images for further processing,
  *   you can set this to true.
+ *
+ * @property {boolean} [connectCache=true] When true, the drawer will reflect changes done to the viewer's
```

Comment:
Maybe one issue with the namig is that I found out the navigator has different cache than the main viewport, which potentially doubles the above-cutoff number of tiles, requests... We could just share the cache within viewers that KNOW they have the same data. Which is the case of navigator. But then this name would be suddently in conflict. So maybe think of a better option name.

Reply:
Sharing the cache between the viewer and the navigator seems like a great optimization. It also sounds like a bigger task. Maybe file an issue so we can track it for later?